### PR TITLE
media-gfx/librecad-1.0.*: Move to EAPI=5

### PR DESCRIPTION
**Pull request by marbre**

### CODE REVIEW

File: media-gfx/librecad/librecad-1.0.4.ebuild [hunk 1]
```diff
@@ -30,13 +29,13 @@ src_unpack() {
 }
```
Comment (jlec) [inline media-gfx/librecad/librecad-1.0.4.ebuild]: missing die

Comment (marbre) [inline media-gfx/librecad/librecad-1.0.4.ebuild]: You're right. Fixed.

**Comment by dxli** [comment]
pushed to repo. Version bump: 2.0.3 => 2.0.7

File: media-gfx/librecad/librecad-1.0.3.ebuild [hunk 2]
```diff
@@ -19,24 +19,23 @@ RDEPEND="
 	dev-qt/qtgui[qt3support]
 	dev-qt/qthelp:4
 	dev-qt/qt3support:4
-	dev-cpp/muParser
-	"
+	dev-cpp/muParser"
 
 DEPEND="${RDEPEND}"
 
 src_unpack() {
```
Comment (junghans) [inline media-gfx/librecad/librecad-1.0.3.ebuild]: Just inherit `vcs-snapshot` and simply drop `src_unpack`.

File: media-gfx/librecad/librecad-1.0.4.ebuild [hunk 3]
```diff
@@ -19,24 +19,23 @@ RDEPEND="
...
 
 src_install() {
-	dobin unix/librecad || die
+	dobin unix/librecad
 	insinto /usr/share/"${PN}"
-	doins -r unix/resources/* || die
+	doins -r unix/resources/*
 	if use doc ; then
 		dohtml -r support/doc/*
```
Comment (junghans) [inline media-gfx/librecad/librecad-1.0.4.ebuild]: You could shorten that to `use doc && dohtml -r support/doc/*`.

**Comment by marbre** [comment]
Done. The merge conflict with commit https://github.com/gentoo-science/sci/commit/238552dc7c3924c446c98b74229a9a6f689fb9bc should only affect the ChangeLog.

**Comment by junghans** [comment]
Could you please also forward port your changes into 2.0.7?

**Comment by marbre** [comment]
Nearly done, just one short question... Replacing the line

*use 3d || sed -i -e '/RS_VECTOR2D/ s/^#//' librecad/src/src.pro*

with

*use 3d || ( sed -i -e '/RS_VECTOR2D/ s/^#//' librecad/src/src.pro || die )*

leads to 
>  ebuild.nesteddie              1
>   media-gfx/librecad/librecad-2.0.7.ebuild: Ebuild calls die in a subshell on line: 30

Howto fix this?

**Comment by dxli** [comment]
this patch works for me:

commit 73075f0127d4ef4176043c965aeb3370dbc52774
Author: Dongxu Li <dongxuli2011@gmail.com>
Date:   Thu Jan 15 14:07:54 2015 -0500

    media-gfx/librecad: fixed USE=3d

diff --git a/media-gfx/librecad/librecad-2.0.7.ebuild b/media-gfx/librecad/librecad-2.0.7.ebuild
index 1700aa8..5a6aa63 100644
--- a/media-gfx/librecad/librecad-2.0.7.ebuild
+++ b/media-gfx/librecad/librecad-2.0.7.ebuild
@@ -32,7 +32,7 @@ src_unpack() {
 
 src_prepare() {
        # currently RS_VECTOR3D causes an internal compiler error on GCC-4.8
-       use 3d || sed -i -e '/RS_VECTOR2D/ s/^#//' librecad/src/src.pro
+       use 3d || ( sed -i'' -e '/RS_VECTOR2D/ s/^#//' librecad/src/src.pro || die )
 }
 
 src_install() {

**Comment by marbre** [comment]
repoman stills complains about the ebuild dieing in a subshell and I don't want to a further QA warning to the overlay.

**Comment by marbre** [comment]
This one should work.

File: media-gfx/librecad/librecad-2.0.7.ebuild [hunk 4]
```diff
@@ -21,18 +21,15 @@ DEPEND="
...
-	mv * ${P}
-}
+RDEPEND="${DEPEND}"
 
 src_prepare() {
 	# currently RS_VECTOR3D causes an internal compiler error on GCC-4.8
-	use 3d || sed -i -e '/RS_VECTOR2D/ s/^#//' librecad/src/src.pro
+	if ! use 3d; then
+		sed -i -e '/RS_VECTOR2D/ s/^#//' librecad/src/src.pro || die
+	fi
```
Comment (junghans) [inline media-gfx/librecad/librecad-2.0.7.ebuild]: Would `use 3d || sed -i -e '/RS_VECTOR2D/ s/^#//' librecad/src/src.pro || die` ok, too?

Comment (marbre) [inline media-gfx/librecad/librecad-2.0.7.ebuild]: Sure. I am always unsure about combined ORs, that why I try to use braces for readability ;)

**Pull request merged**